frontend/src/components/form/api/ApiWrapper.vue

```diff
@@ -177,6 +178,12 @@ export default {
         this.debouncedSave()
       }
     },
+    async onBlur() {
+      if (!this.isSaving && this.autoSave) {
```

Comment:
```suggestion
      if (!this.dirty) {
```

If the field is exited - and it is not dirty, then the current `apiValue` may be accepted. 
Effect: If Api has removed spaces, we remove them as soon as the field is exited - but not before. As long as the user is in the input field, he is in control. As soon as he leaves it, we show what we have done with his input.

Reply:
I needed to add an additional field, that is true until the debounced save starts. Dirty will still be true, in the case of trimming, because the local value and the api value will never be the same.